Geometry_Engine/Query/Centre.cs

```diff
@@ -40,6 +39,7 @@ public static Point Centre(this Arc arc, double tolerance = Tolerance.Distance)
 
         /***************************************************/
 
+        [ToBeRemoved("4.1", "To be removed as it is generally incorrect. Advising using Centroid instead.")]
```

Comment:
Could we do some versioning so that scripts automatically change from `centre` to `centroid`? @IsakNaslundBh what do you think in terms of UX on that?

Reply:
Wouldn't `ReplacedAttribute` do the trick here?

Reply:
I think it would, do you agree we need it? I'm happy with a `ToBeRemoved` but I'm wondering if changing to `centroid` or `average` is a better UX is all. Maybe `average` as that's closest to what the original method was doing? Though `centroid` might be more related to what people _thought_ it was doing?

Reply:
I actually think keeping it as is, with the `ToBeRemoved`, makes most sense. Given, as you stated above, that this functionally could be replaced with average of controlpoints, and semantically with the centroid, I think simply deprecating it (with ToBeRemoved) seems like the right thing to do.

Adding versioning so that Centroid was called instead of centre would ofc be trivial from the code side, but would mean a functional change that I am not sure users would want.

TL;DR I think what Lukasz has done makes sense.

Reply:
Ok, SGTM, marking as resolved, approving and merging :smile: